### DISCUSSION TITLE
Is it necessary to create your own attestation service?

### DISCUSSION THREAD

**Discussion by kirill-serkin** [celo-org/social-connect#244] [Help (Q&A)]
Hi, I am a blockchain developer. Just recently decided to start developing with Celo. The possibility of sending tokens by phone number to which a blockchain wallet is tied attracts a lot of attention. Hence, I have a question, is it necessary to create my own attestation service?
Are there any tools that work in test mode?

Thank you in advance!

**Comment by arthurgousset** [comment]
Hey, thanks for reaching out here! 

To use SocialConnect, you don't need to run an attestation service that sends SMS.
You can simply register a mapping without performing any verification 

Take a look at the NodeJS examples using your preferred client library:
- [ethersjs](https://github.com/celo-org/SocialConnect/blob/main/examples/ethers.ts)
- [web3js](https://github.com/celo-org/SocialConnect/blob/main/examples/web3.ts)
- [@celo/contractkit](https://github.com/celo-org/SocialConnect/blob/main/examples/contractKit.ts)

Please let @therealharpaljadeja know if you run into any issues. We're super happy to help! :)
